File: Java/Ex_estrutura_condicional/ex2 soma impares/soma_impares.java
/*Exemplo 1: 
Digite dois numeros: 
2 
9
SOMA DOS IMPARES = 15  */

import java.util.Scanner; 
import java.util.Locale;  

public class soma_impares {
    public static void main (String[] args) {
        
        Locale.setDefault(Locale.US); 
        Scanner sc = new Scanner(System.in);

        int num1, num2, troca, soma = 0;

        System.out.println("Digite dois numeros: ");
        num1 = sc.nextInt();
        num2 = sc.nextInt();

        if (num1 > num2) {
            troca = num1;
            num1 = num2;
            num2 = troca;
        }
        for (int i = num1 + 1; i < num2; i++) {
            if (i % 2 != 0) {
                soma = soma + i;
            }
        }
        System.out.println("SOMA DOS IMPARES = " + soma);
        
        sc.close();
    }
}

File: Python/Ex estrutura repetitivas/ex2 soma impares/soma_impares.py
"""
Exemplo 1: 
Digite dois numeros: 
2 
9
SOMA DOS IMPARES = 15 
"""

print("Digite dois numeros: ")
num1 = int(input())
num2 = int(input())

if num1 > num2:
    troca = num1
    num1 = num2
    num2 = troca

soma = 0
for i in range(num1 + 1, num2):
    if i % 2 != 0 :
        soma = soma + i

print(f"SOMA DOS IMPARES = {soma}")
